drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.h

```diff
@@ -1090,5 +1090,7 @@ int mt76_connac_mcu_set_roc(struct mt76_dev *dev, struct ieee80211_vif *vif,
 void mt76_connac_mcu_roc_event(struct mt76_phy *phy,
 			       struct mt76_connac_roc *roc,
 			       struct mt76_connac_roc_tlv *event);
-
+int mt76_connac_sta_state_dp(struct mt76_dev *dev,
```

Comment:
@objelf I think this will not apply upstream :)

Reply:
do you mean it is not neccessary to remove the blank line, right ?

Reply:
@objelf I mean mt76_connac_mcu_roc_event is not upstream yet, but no worries I will rebase it, just send me an updated PR on top of mt7921e-devel branch

Reply:
ok, thx